package/yast2-services-manager.spec

```diff
@@ -24,16 +24,19 @@
 ######################################################################
 
 Name:           yast2-services-manager
-Version:        4.1.4
+Version:        4.1.5
 Release:        0
 BuildArch:      noarch
 
 BuildRoot:      %{_tmppath}/%{name}-build
 Source0:        %{name}-%{version}.tar.bz2
 
 Requires:       ruby
-# Yast2::SystemService#found?
-Requires:       yast2 >= 4.0.83
+
+# Yast2::Systemd::Target
+BuildRequires:  yast2 >= 4.1.3
+Requires:       yast2 >= 4.1.3
```

Comment:
NP: In this spec, Requires and BuildRequires are separated. I would keep it as it was.